core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2432,6 +2432,10 @@ private[spark] object Utils extends Logging {
   }
 }
 
+private[util] object CallerContext {
+  var callerContextSupported: Boolean = true
```

Comment:
What is the usage of this flag? I don't see any other place use it, all just setters.